Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -96,6 +93,7 @@ def fetch_indicators_command(
             last_run_ctx[collection.id] = client.last_fetched_indicator__modified
     else:
         # fetch from a single collection
+        filter_args["added_after"] = get_added_after(fetch_full_feed, initial_interval, last_fetch_time)
```

Comment:
why was this move inside the `else` scope?

Reply:
Because it was deleted from before the statement,
And if the condition is true then this value is calculated in each iteration so there is no reason to calculate it before.